Tests/Data/Mechanics/Linear/Python/gen-unit-circle.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/vtkpython
+
+import sys
+import numpy as np
+from vtk.util.numpy_support import vtk_to_numpy, numpy_to_vtk
+import vtk
+
+in_grid, out_grid, out_geom = sys.argv[1:]
+
+reader = vtk.vtkXMLUnstructuredGridReader()
+reader.SetFileName(in_grid)
+reader.Update()
+
+grid = reader.GetOutput()
+assert grid.GetBounds() == (0.0, 1.0, 0.0, 1.0, 0.0, 0.0)
+
+coords = vtk_to_numpy(grid.GetPoints().GetData())
+
+a = np.empty(coords.shape[0])
+
+for i, (x, y, z) in enumerate(coords):
+    r = np.sqrt(x**2 + y**2)
```

Comment:
Where is `r` used?

Reply:
In some old version of this script, long, long time ago. I'll remove it.